release/linux-specific/linux-install.sh

```diff
@@ -109,31 +132,17 @@ else
 
 fi
 
-## symlink just installed version to /opt/omegat/OmegaT-default
+## symlink /opt/omegat/$OMTVERSION (version just installed) to /opt/omegat/OmegaT-default (default directory)
 
 sudo ln -s -b -T /opt/omegat/$OMTVERSION /opt/omegat/OmegaT-default
 
 # symlink bash OmegaT launch script
 # from <OmegaT version> to /usr/local/bin
+# copy omegat.desktop configuration file 
+# to /usr/share/applications/
 
 sudo ln -s -b /opt/omegat/OmegaT-default/OmegaT /usr/local/bin/omegat
 
-# symlink Kaptain OmegaT launch script
-# from <OmegaT version> to /usr/local/bin
-
-sudo ln -s -b /opt/omegat/OmegaT-default/omegat.kaptn /usr/local/bin/omegat.kaptn
-
-sudo chmod +x /usr/local/bin/omegat /usr/local/bin/omegat.kaptn
-
-# install icons
-icon_sizes=( 32 128 256 512 )
-for size in "${icon_sizes[@]}"
-do
-	sudo xdg-icon-resource install --novendor --noupdate --mode system --size $size /opt/omegat/OmegaT-default/images/OmegaT.iconset/icon_$size\x$size.png omegat
-done
-sudo xdg-icon-resource forceupdate --mode system
-
-# install desktop file
-sudo xdg-desktop-menu install --novendor --mode system /opt/omegat/OmegaT-default/omegat.desktop
+sudo cp omegat.desktop /usr/share/applications/
```

Comment:
It should use xdg for standard compliance. 
This removes standard compliant process. I'm against it.